New version of conf-patcher written in python

seehausen opened the pull request:
the new python-script `patch_conf.py` helps you to easily adapt your personal changes in the skin.conf file.

Create a patchfile called `skin.conf.patch` and edit it to with the values you want to be changed. take a look into the `skin.conf.patch.sample`

With the patch script you can easily update to the newest version of the skin, and than let the script do the changes to your personal setting

the patch file has the same structure as the skin file,
just add the values you want to be changed

to run the script call 
`python3 patch_conf.py  `

Example
```
[Extras]
  [[Header]]
    custom1_label = Homepage
    custom1_url = https://myhomepage.org/

  [[Footer]]
    name = station Name
    link1_label = Lightning Maps
    link1_url = https://www.lightningmaps.org/#m=ses;t=3;s=0;o=0;b=;ts=0;y=48.7750;x=11.4550;z=8;d=2;dl=2;dc=0;

  [[Appearance]]
    embedded_order = iFrame_windy

  [[Embedded]]
     [[[iFrame_windy]]]
       url = "https://embed.windy.com/embed.html?type=map&location=coordinates&metricRain=default&metricTemp=default&metricWind=default&zoom=5&overlay=wind&product=ecmwf&level=surface&lat=48.717&lon=11.495"
       title = Windy Map
```

tfilo commented on install.py:
It would be good to add patch_conf.py here so it get installed with skin

seehausen replied:
added

tfilo commented:
Other than mentioned battery section it works well. Maybe is it possible to commit it with "x" permision so it is possible to run like ./patch_conf.py without need to change permisions at first ?

seehausen commented:
@tfilo 
Can you check again, now it should also patch or add 4. level sections
also changed install.py and removed old patch variant

patch_conf.py should be exectutable. It had +x attribute already, not sure why it was not there

seehausen commented on skins/neowx-material/patch_conf.py:
should be fixed

seehausen commented:
Sorry ... needs to be reworked :-(

seehausen commented:
OK, hopefully fixed all cases

seehausen commented:
ah that could be the reason, I run the patch on my mac not on the raspi... I will check
